.env.example

```diff
@@ -6,6 +6,7 @@ APP_TIMEZONE=America/New_York
 APP_CLEAR_TASKLOG=720
 APP_DELETE_MINUTES=10
 APP_ENVIRONMENT_ONLY=true
+LANG_MUST_BE_GLOBAL=false
```

Comment:
maybe call this `APP_LOCALE_IS_GLOBAL=false`

Reply:
Done